### PR TITLE
Remove unused expectedRV in store_test TestGet

### PR DESCRIPTION
**Pull request by wenjiaswe**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
expectedRV is defined in tests struct  but not set in test cases, removing the unnecessary checking

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go [hunk 1]
```diff
@@ -226,7 +226,6 @@ func TestGet(t *testing.T) {
 		expectRVTooLarge  bool
 		expectedOut       *example.Pod
 		rv                string
-		expectedRV        string
```
Comment (jingyih) [inline staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go:229]: field `expectedRV` might be related to the TODO(jpbetz) on line 220? I will defer to @jpbetz.

Comment (jpbetz) [reply staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go:229]: It's fine to remove `expectedRV`. I'm pretty sure there was a WIP version of this where I wanted to test exact RVs and then later I decided to take a more black box approach.

**Comment by jingyih** [comment]
/lgtm

/assign @wojtek-t
for approval

**Pull request merged**